### ISSUE TITLE
wav2vec finetune on large scale labeled data

### ISSUE THREAD

**Issue by lixx0105** [facebookresearch/fairseq#4017]
### Before asking:
1. search the issues.
2. search the docs.

#### What's your environment?

 - fairseq Version (e.g., 1.0 or main):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by lixx0105** [comment]
hi @alexeib，may i have your advice on wav2vec finetune on large scale label data(10k hour)? It seem finetune is hard. For example, I got 20% cer when I finetune on 1k hour data(data A), but can only get 30% cer when I finetune on 10k hour data which include data A. if I finetune on data A first and then continue to finetune on 10k hour, I got slightly better cer than 20%. Can I have your advice? Thanks
